src/register/Register.jsx

```diff
@@ -21,8 +21,13 @@ export default function Register() {
                     payload[key] = false;
             else
                 payload[key] = value;
-            });
+        });
 
+        if (!("available_to_mentor" in payload))
+            payload["available_to_mentor"] = false;
+        if (!("need_mentoring" in payload)) 
```

Comment:
the code in both of these is almost the same. Do you see scope for refactoring? instead of duplicating the code?

Reply:
I can't see how this is duplicating since there are 2 attributes here that I check and add if not in payload, "need_mentoring" and "available_to_mentor". Maybe tonight you can give me suggestion how to make it more efficient without assigning it to it's exact attribute.